vnext/ReactUWP/ReactUWP.vcxproj

```diff
@@ -418,8 +418,8 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets')" />
-    <Import Project="$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.3\build\native\ReactNative.Hermes.Windows.targets" Condition="Exists('$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.3\build\native\ReactNative.Hermes.Windows.targets') AND '$(OSS_RN)' != 'true' AND '$(USE_HERMES)' == 'true'" />
-    <Import Project="$(SolutionDir)packages\ReactNative.V8JSI.Windows.0.1.4\build\native\ReactNative.V8JSI.Windows.targets" Condition="Exists('$(SolutionDir)packages\ReactNative.V8JSI.Windows.0.1.4\build\native\ReactNative.V8JSI.Windows.targets') AND '$(OSS_RN)' != 'true' AND '$(USE_V8)' == 'true'" />
+    <Import Project="$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.6\build\native\ReactNative.Hermes.Windows.targets" Condition="Exists('$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.6\build\native\ReactNative.Hermes.Windows.targets') AND '$(OSS_RN)' != 'true' AND '$(USE_HERMES)' == 'true'" />
```

Comment:
This is the latest and best version .. and I was thinking we are already on 0.1.6 .. Thanks !

Reply:
Actually, you should reference V8/Hermes package like we do in https://github.com/microsoft/react-native-windows/blob/master/vnext/ReactWindowsCore/ReactWindowsCore.vcxproj ..

My understanding was that, as ReactUWP links to ReactWindowsCore, these lines are not needed in ReactUWP ..

Reply:
The common variable are defined in https://github.com/microsoft/react-native-windows/blob/master/vnext/PropertySheets/React.Cpp.props

Reply:
ok. I removed them from ReactUWP project.